File: practice/mathematics/fundamentals/handshake/Handshake.scala
import scala.io.Source

object Handshake extends App {
  val lines = Source.stdin.getLines()
  val directors = lines.drop(1).map(_.toInt)
  val handshakes = directors.map(Range(0, _).toList.sum)
  println(handshakes.mkString("\n"))
}

File: practice/mathematics/fundamentals/handshake/Handshake.java
import java.util.*;

public class Handshake {

    public static void main(String[] args) {
        Scanner stdin = new Scanner(System.in);
        int tests = stdin.nextInt();
        for(int i = 0; i < tests; i++) {
            long handshakes = 0;
            int directors = stdin.nextInt();
            for(int j = 0; j < directors; j++) {
                handshakes += j;
            }
            System.out.println(handshakes);
        }
        stdin.close();
    }
}
